assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/jgroups/che-tcp.xml

```diff
@@ -21,29 +21,30 @@
     send_buf_size="1M"
     max_bundle_size="64K"
     enable_diagnostics="true"
+    log_discard_msgs="false"
     thread_naming_pattern="cl"
 
     thread_pool.min_threads="0"
     thread_pool.max_threads="500"
     thread_pool.keep_alive_time="30000" />
 
-  <kubernetes.KUBE_PING
-    labels="app=che" />
+  <org.jgroups.protocols.kubernetes.KUBE_PING
```

Comment:
What the source of this file? Does it come from documentation

Reply:
Yes that's exacly as in doc on https://github.com/jgroups-extras/jgroups-kubernetes